blscurve.nimble

```diff
@@ -33,3 +33,11 @@ task test, "Run all tests":
 
   # Public BLS API - IETF standard / post Ethereum2.0 v0.10
   test "-d:BLS_USE_IETF_API=true", "tests/eth2_vectors.nim"
+
```

Comment:
should also get compiled by CI

Reply:
Added in https://github.com/status-im/nim-blscurve/commit/02e8216e3ea553b454ec5b1852e1a51b2999f2c5